src/workerd/io/worker.h

```diff
@@ -736,6 +736,11 @@ class Worker::Actor final: public kj::Refcounted {
   // Runs an alarm in this actor. This function will handle deduplicating multiple alarms. If this
   // isn't a duplicate alarm, func will be called to run the alarm.
 
+  void unsetRequestTracker() {
```

Comment:
I think there is a `TODO(soon)` below that we probably want to remove since we're relying on the maybeness.